src/client/proxy/handlers/common.ts

```diff
@@ -198,9 +202,11 @@ const getHttpRequestHandler = () => async (req: FastifyRequest, res: FastifyRepl
                 const templateFileContents = await getFile(templateFileId);
                 const renderer = new Renderer({rootDirId} as any);
 
-                res.header('Content-Type', 'text/html');
+                const contentType = ext ? getContentTypeFromExt(ext) : 'text/html';
```

Comment:
Can template file be something except HTML?

Reply:
Yes, in google font mirror I render CSS on the fly via zhtml, but then the browser refused to consume it due to content-type being text/html and not text/css